src/Scaler/Services/ScalerMetadata.cs

```diff
@@ -9,10 +9,10 @@ internal sealed class ScalerMetadata
     {
         private string _metricName;
 
-        public string Connection { get; set; }
+        public string ConnectionFromEnv { get; set; }
```

Comment:
Changes look good but wanted to have a discussion.

I think we should keep the property name same as before and add attribute `[JsonProperty("ConnectionFromEnv")]` on them. We don't need to keep reminding from where these properties arrived in the code. Also, the one in scaled object and in code are essentially different, because the scaled object points to the environment variable name but here, it's pointing to connection string literal.

Reply:
Agree.

Reply:
Makes sense, updated it. Please review the changes.